src/pages/SetItems.jsx

```diff
@@ -56,36 +99,54 @@ const SetItems = () => {
 		]
 	}
 
-
 	return (
 		<IonPage>
 			<IonContent scrollEvents={false}>
-				<IonToolbar style={{ marginTop: 10, paddingLeft: 10}}>
-					<IonButtons style={{display: "inline-block"}} onClick={() => history.push("/home")} >
-						<IonIcon icon={arrowBackOutline} style={{ fontSize: 30, color: "gray"}}/>
-					</IonButtons>
-					<div style={{display: "inline-block", marginLeft: 10, maxWidth: "85%"}}>
-						<IonCardTitle style={{fontSize: "1.2em"}}>Set of Cards</IonCardTitle>
-						<IonCardSubtitle style={{fontWeight: "normal", textTransform: "inherit"}}>
-							Available Cards
-						</IonCardSubtitle>
-					</div>
-				</IonToolbar>
+				<IonHeader>
+					<IonToolbar>
+						<IonButtons slot="start" style={{paddingLeft: "5px"}}>
+							<IonBackButton defaultHref="home" text="" icon={arrowBackOutline} style={{color: "#7D7D7D"}} />
+						</IonButtons>
+						
+						<div slot="secondary">
+							<Indicator  label="10"/>
+							<ProgressBar label="MemPoints: 10/100"/>
+						</div>
+						
+						<IonFabButton slot="end" style={{"--background": "none", boxShadow: "none", "--border-color": "none", "--box-shadow": "none", width: "25px", height:"25px"}}>
+							<IonIcon icon={trashOutline} style={{color:"#575757"}} onClick={() => setPromptVisible(true)}/>
+						</IonFabButton>
+					</IonToolbar>
+				</IonHeader>
 				<IonGrid>
 					<IonRow >
-						<RenderItems/>
+						{/* <RenderItems/> */}
+						<Item/>
+						{/* Add New Card Button */}
+						<IonCol size="6">
+							<IonCard style={{boxShadow: "none", margin: 0, paddingBottom: "152%"}}>
+								<div className="" style={{ width: "100%", borderRadius: "5px", display: "inline-block", height: "100%", border: "4px dashed #B7B0FF", position: "absolute"}}>
+									<IonCardContent className="container">
+										<IonCardTitle style={{color: "#B7B0FF"}}>
+											<IonIcon icon={addSharp} style={{width: "50px", height: "50px"}}/>
+											<h1 style={{fontSize: "13px", lineHeight: "15px"}}>Add New Card</h1>
+										</IonCardTitle>
+									</IonCardContent>
+								</div>
+							</IonCard>
+						</IonCol>
```

Comment:
Put on click functionality to this, that will redirect you to create an item page.